protobuf/execapi/api.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+
+import "protobuf/definition/execution.proto";
+
+package api;
+option go_package = "github.com/mesg-foundation/core/protobuf/execapi";
+
+service ExecutionService {
```

Comment:
I'm ok with that if we had only the ExecutionService but we will have the service so end up with:
```
service ServiceService {
```
I'm ok to keep just `Execution` here and anyway it will be in a different package so `execapi.Execution` and `definition.Execution`